ui/widgets/nuxeo-tag-suggestion.js

```diff
@@ -243,10 +243,23 @@ import { escapeHTML } from './nuxeo-selectivity.js';
     _addedTagHandler(entry) {
       if (this.document) {
         this.$.addTagOp.params = { tags: entry.id };
-        this.$.addTagOp.execute().then(() => {
+        this.$.addTagOp.execute().then((response) => {
+          const lenArr = response.properties['nxtag:tags'].length;
+          const { label } = response.properties['nxtag:tags'][lenArr - 1];
```

Comment:
what happens if response.properties['nxtag:tags'] is empty array. In that case this line will throw an exception. Add safe checks